ui/components/ui/snackbar/index.scss

```
@@ -2,10 +2,10 @@
   @include H7;
 
   padding: 0.75rem 1rem;
-  color: var(--Blue-600);
+  color: var(--color-text-default);
```

```suggestion
  color: var(--color-primary-alternative);
```

If we want to stay with the original blue text

---

So for alerts we are using `--color-text-default` this is correct! Sorry I should have notified the DM team